samples/wifi/monitor/src/main.c

```diff
@@ -66,6 +71,13 @@ typedef struct {
 	int qosDataCount;
 	int nullCount;
 	int qosNullCount;
+	int actionCount;
+	int cfEndCount;
+	int unknownMgmtCount;
+	int unknownCtrlCount;
+	int unknownDataCount;
+	int reservedFrameCount;
+	int unknownFrameCount;
```

Comment:
Why does it not follow the coding style?

Reply:
Do you mean not using the camel case?

Reply:
yes

Reply:
I couldn't find anything in confluence or Zephyr guidelines that camel case is disallowed, can you please point me to any link for guidelines?

Reply:
You are probably familiar with the code base and have noticed that, with very few exceptions, no camelcase is used.
https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style
https://kernel.org/doc/html/latest/process/coding-style.html
https://kernel.org/doc/html/latest/process/coding-style.html#naming

Reply:
Sure, and I did go through above Zephyr links, but couldn't find any reference to `camel case`. I know its frowned upon in Linux, but wasn't sure about Zephyr.